htdocs/themes/math4/math4.scss

```diff
@@ -301,6 +303,11 @@ $site-nav-width: 250px !default;
 	}
 }
 
+hr.hr {
+	margin: 3pt 7pt;
+	border: 1pt solid;
+}
+
```

Comment:
I think it would be best to use a class name that is not just the tag name.  Perhaps `site-nav-separator`.  Also, unless you have plans to use this elsewhere, this should be moved inside the `#site-navigation` section.